lib/teams/index.ts

```diff
@@ -28,6 +28,11 @@ export class TeamProps {
      */
     readonly namespaceAnnotations? : { [key: string]: any; } = { "argocd.argoproj.io/sync-wave": "-1" };
 
+    /**
+     * Labels such as necessary for AWS AppMesh 
+     */
+    readonly namespaceLabels? : { [key: string]: any; } = {"appmesh.k8s.aws/sidecarInjectorWebhook": "enabled"};
```

Comment:
Let's not assign this default to the namespace. We should have labels, but the content should be provided by the customers when they define their teams.

Reply:
Yes 100% agree, removed as suggested.